tools/dds/dds-server/lrs-device-controller.h

```diff
@@ -3,31 +3,32 @@
 
 #pragma once
 #include <librealsense2/rs.hpp>  // Include RealSense Cross Platform API
+#include <librealsense2/utilities/json.h>
```

Comment:
Any reason cannot just use the `json_fwd.hpp`?

Reply:
Changed